src/toggle/item/ToggleItem.tsx

```diff
@@ -28,12 +28,38 @@ function ToggleItem({
     "toggle-item--is-disabled": isDisabled
   });
 
+  if (canSelectMultiple) {
+    return (
+      <ListItem
+        testid={testid}
+        customClassName={toggleItemClassName}
+        clickableListItemProps={isDisabled ? undefined : {onClick: handleToggle}}>
+        <label className={"toggle-item__label"}>
+          <input
+            type={"checkbox"}
+            className={"toggle-checkbox-input"}
+            checked={isSelected}
+            disabled={isDisabled}
+          />
+          {children}
+        </label>
+      </ListItem>
+    );
+  }
   return (
     <ListItem
       testid={testid}
       customClassName={toggleItemClassName}
       clickableListItemProps={isDisabled ? undefined : {onClick: handleToggle}}>
-      {children}
+      <label className={"toggle-item__label"}>
+        <input
+          type={"radio"}
+          className={"toggle-radio-input"}
+          checked={isSelected}
+          disabled={isDisabled}
+        />
+        {children}
+      </label>
```

Comment:
If we don't need more different changes let's just change the input type. Also, we should have only one `return` inside the components. Otherwise, it would be confusing.

```suggestion
      <label className={"toggle-item__label"}>
        <input
          type={canSelectMultiple ? "checkbox" : "radio"}
          className={"toggle-radio-input"}
          checked={isSelected}
          disabled={isDisabled}
        />
        {children}
      </label>
```